assets/ubuntu-welcome.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Initial Setup
```

Comment:
Ubuntu Welcome?

Reply:
Perhaps... but we already have translations for Initial Setup and actually it isn't displayed anywhere.  I can change it